### ISSUE TITLE
Ability to embed a profile or timeline

### ISSUE THREAD

**Issue by themushmouth** [tootsuite/mastodon#6094]
Is it possible to embed a profile or timeline? I would like to embed my mastodon profile to my site.

**Comment by sorin-davidoi** [comment]
No, only individual posts are embeddable right now.

**Comment by schiessle** [comment]
You could do this by parsing the RSS feed, that's what I did to add my toot's to my webpage. Maybe it is useful for you https://gitlab.com/BeS/hugo-sustain-ng/blob/master/static/js/feedreader.js

**Comment by themushmouth** [comment]
@schiessle what rss feed did you use?

**Comment by schiessle** [comment]
@themushmouth afaik, there is only one per user: https://mastodon.social/@bjoern.atom

**Comment by FedericoCeratto** [comment]
See #1065 for embedding individual post.

**Comment by poVoq** [comment]
@schiessle Your current website also seems to get various other parts from Mastodon now. Is that still done via the RSS feed? Would it be very difficult to abstract that part out to use in any static website?

**Comment by schiessle** [comment]
no, I wrote a small PHP app which uses the mastodon API to fetch all the information: https://gitlab.com/BeS/hugo-sustain-ng/tree/master/static/mastodon.comments

**Comment by idotj** [comment]
After checking several pages/forums I decided to create an embed timeline with only html and js (no jQuery dependency).
Here you have the source code:
https://gitlab.com/idotj/mastodon-embed-feed-timeline

**Comment by appel** [comment]
> https://mastodon.social/@bjoern.atom

It appears the URL format has changed from .atom to .rss:
https://mastodon.social/@bjoern.rss
